tests/unit/cli/admin/test_sign.py

```diff
@@ -14,11 +14,18 @@
 
 
 class TestSign:
-    def test_sign_with_previous_root(self, test_context, patch_getpass):
+    def test_sign_with_previous_root(
+        self, monkeypatch, test_context, patch_getpass
+    ):
         inputs = [
-            "1",  # Please enter signing key index
             f"{_PEMS / 'JH.ed25519'}",  # Please enter path to encrypted private key  # noqa
         ]
+        # selections interface
+        monkeypatch.setattr(
+            "repository_service_tuf.cli.admin.helpers._select",
```

Comment:
Use https://github.com/repository-service-tuf/repository-service-tuf-cli/blob/bfbf0a545024d9e72c533bb2a7318c26e5e7a123/tests/conftest.py#L45 to shorten paths.